src/main/java/org/apache/maven/report/projectinfo/SummaryReport.java

```diff
@@ -148,22 +148,14 @@ private String getMinimumJavaVersion() {
             String sourceConfigured = getPluginParameter(pluginId, "source");
             String targetConfigured = getPluginParameter(pluginId, "target");
 
-            String forkFlag = getPluginParameter(pluginId, "fork");
-            String compilerVersionConfigured = null;
-            if ("true".equalsIgnoreCase(forkFlag)) {
-                compilerVersionConfigured = getPluginParameter(pluginId, "compilerVersion");
-            }
-
-            String minimumJavaVersion = compilerVersionConfigured;
```

Comment:
Is it not needed?

Reply:
No. it's not. It's set to a different value on all three branches.

Reply:
Unfortunately yes, but to me it looks like bug

Reply:
Perhaps. Do you see a case where it should be set to this?

Reply:
Maybe we should combine the ifs so 

if (targetConfigured != null)

becomes 

else if if (targetConfigured != null)